src/java/org/apache/cassandra/config/Config.java

```diff
@@ -160,6 +160,8 @@ public static Set<String> splitCommaDelimited(String src)
     @Replaces(oldName = "slow_query_log_timeout_in_ms", converter = Converters.MILLIS_DURATION_LONG, deprecated = true)
     public volatile DurationSpec.LongMillisecondsBound slow_query_log_timeout = new DurationSpec.LongMillisecondsBound("500ms");
 
+    public volatile DurationSpec.LongMillisecondsBound timeout_delay = new DurationSpec.LongMillisecondsBound("43000000ms");
```

Comment:
Can we name this more specifically as there are many timeouts in Cassandra - e.g. `stream_transfer_task_timeout` and maybe the timeout could be a more readable `12h`?